install/0000_90_cluster-version-operator_02_servicemonitor.yaml

```diff
@@ -127,3 +127,12 @@ spec:
       for: 10m
       labels:
         severity: warning
+    - alert: CannotEvaluateConditionalUpdates
+      annotations:
+        summary: Cluster Version Operator cannot evaluate conditional update matches for {{ "{{ $value | humanizeDuration }}" }}.
+        description: Failure to evaluate conditional update matches means that Cluster Version Operator cannot decide whether an update path is recommended or not.
+      expr: |
+        (cluster_version_conditional_updates_recommended_conditions_seconds{recommended="unknown"}) >= 3000
```

Comment:
you might want a `max by (version, ...)` or or some such to filter out  things that are not relevant in the resulting alert, like `instance="..."` and `pod="cluster-version-operator-..."`.  Not a big deal to keep them around, but if you do, a relocating CVO could trigger one alert to close and moments later a new alert with labels from the new CVO to open, which is slightly distracting for folks ack'ing alerts in PagerDuty and such.

Reply:
Thanks, I was not aware of this, very useful point.